src/components/transaction-item.tsx

```diff
@@ -158,6 +159,19 @@ const Timestamp: React.FC<BoxProps & { tx: Transaction | MempoolTransaction }> =
   }
 );
 
+const Nonce: React.FC<BoxProps & { nonce: Number, isHovered?: boolean }> = React.memo(
```

Comment:
Does this component only need `nonce`? 
```suggestion
const Nonce: React.FC<{ nonce: Number }> = React.memo(
```

Reply:
Probably, I'm new to React so I was really winging this PR.